### PR TITLE
feat(replays): Force a 'Replay Start' breadcrumb in the Chapters list

### LINKED ISSUE #34693: Investigate whether the Chapters component is allowed to be empty (and needs to render an empty state)

### PR DESCRIPTION
**Pull request by ryan953**
Sticking a new 'Replay Start' breadcrumb to the top of the Chapters list.

Screen Shot 2022-05-19 at 12 58 55 AM

Fixes #34693

### CODE REVIEW

File: static/app/utils/replays/replayDataUtils.tsx [hunk 1]
```diff
@@ -56,12 +57,19 @@ export function breadcrumbValuesFromEvents(events: Event[]) {
...
 
-export function breadcrumbEntryFactory(_startTimestamp: number, rawCrumbs: RawCrumb[]) {
-  // TODO: insert init breadcrumb
+export function breadcrumbEntryFactory(startTimestamp: number, rawCrumbs: RawCrumb[]) {
+  const initBreadcrumb = {
+    type: BreadcrumbType.SESSION,
+    timestamp: new Date(startTimestamp).toISOString(),
+    level: BreadcrumbLevelType.INFO,
+    action: 'replay-init',
+    message: 'Start recording',
```
Comment (billyvg) [inline static/app/utils/replays/replayDataUtils.tsx:66]: may need to import

```suggestion
    message: t('Start recording'),
```

**Review by billyvg — APPROVED** [review]

File: static/app/components/replays/actionCategory.tsx [hunk 2]
```diff
@@ -34,6 +34,11 @@ function getActionCategoryInfo(crumb: Crumb): ActionCategoryInfo {
         title: t('Error'),
         description: `${crumb.category}: ${crumb.message}`,
       };
+    case BreadcrumbType.SESSION:
+      return {
+        title: t('Replay Start'),
+        description: crumb.data?.url,
+      };
```
Comment (ryan953) [inline static/app/components/replays/actionCategory.tsx:41]: slightly re-purposed `SESSION` and making the assumption that it's our injected start crumb.

Comment (billyvg) [reply static/app/components/replays/actionCategory.tsx:41]: Would it be better to create a new type?

Comment (ryan953) [reply static/app/components/replays/actionCategory.tsx:41]: `BreadcrumbType.INIT`

**Pull request merged**